### PR TITLE
[11.0] [MIG] base_view_inheritance_extension: Migration to v11.0

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa
WIP

### CODE REVIEW

**Review by pedrobaeza — COMMENTED** [review]
Please squash adjacent transbot commits + pre-generate README

**Comment by sergio-teruel** [comment]
@pedrobaeza done!

File: base_view_inheritance_extension/__init__.py
Comment (NL66278) [inline base_view_inheritance_extension/__init__.py]: Copyright can/should be removed from __init__.py files.

File: base_view_inheritance_extension/__manifest__.py
Comment (NL66278) [inline base_view_inheritance_extension/__manifest__.py]: Both copyright for Therp BV and license stuff can/should be changed to https. Copyright symbol should be replaced by the word Copyright.

**Review by NL66278 — CHANGES_REQUESTED** [review]
Some minor style improvements needed before approval. See other comments.

**Review by cubells — APPROVED** [review]
If you do changes requested :+1: 

LGTM

**Comment by NL66278** [comment]
@sergio-teruel Are you going to take up the review comments?

**Comment by pedrobaeza** [comment]
@NL66278 I have just made the requested changes.

**Review by NL66278 — APPROVED** [review]
:+1:  Comments done and LGTM

**Pull request merged**
